guide/creating_element/testing.md

```diff
@@ -144,6 +144,55 @@ end
 
 Once again, we are comparing the pipeline's output with our reference file to assert that everything works correctly.
 
+### Customizing pipeline module
+
+[`Membrane.Testing.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html) allows us to customize every callback of [`Membrane.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Pipeline.html). Let's create Pipeline that will answer to requests with response we passed in initialization options. To do this, we must define our custom module.
+
+```elixir
+defmodule Example.Pipeline do
+  use Membrane.Pipeline
+  
+  def handle_init(options) do
+    state = %{response: options.response}
+    spec = %Membrane.Pipeline.Spec{children: [], links: []}
+    {{:ok, spec}, state}
+  end
+  
+  def handle_other({:request, from}, state) do
+    send(from, {:response, state.response})
+    {:ok, state}
+  end
+  
+  def handle_other(_message, state),
+    do: {:ok, state}
+end
+```
+Note that `Membrane.Pipeline.Spec` of our pipeline module can contain empty lists of children and links. 
+In order to use callbacks defined in `MockPipeline` and pass options as , we have to specify it in `Options`. 
+
+```elixir
+Pipeline.start_link(%Pipeline.Options{
+  ...
+  module: Example.Pipeline,
+  custom_args: %{response: "Hello there!"}
+})
+```
+
+Let's create another test case to try out new functionality.
```

Comment:
`Let's` create is used twice in this article, maybe it could replaced with something like `in order to try out....`